topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,640 @@
+time_estimation: "1h"
+key_points:
+  - "Use dataset collections"
+  - "With SearchGUI and PeptideShaker you can gain access to multiple search engines"
+  - "Learning the basics of SQL queries can pay off"
+contributors:
+  - subinamehta
+  - timothygriffin
+  - pratikdjagtap 
+  - emmaleith
+  - mariecrane
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+Metaproteomics involves characterization of community level expression of microbial proteins from an environmental 
+or clinical sample. Metaproteomics data is primarily used to determine the functional status of the microbiome under 
+study along with its taxonomic composition. The Galaxy-P team published a software suite named [metaQuantome](https://www.mcponline.org/content/18/8_suppl_1/S82) to enable quantitative and statistical analysis and visualization of functional, taxonomic expression as well as functional and 
+taxonomy interaction. metaQuantome leverages peptide level quantitative information to analyze the taxonomic, functional 
+expression within the microbial community in different conditions.
+
+![Microbiome](../../images/microbiome.png){: width="75%"}
+ 
+metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, 
+across multiple experimental conditions. metaQuantome, an open source tool, is available via command line and also 
+accessible via Galaxy platform for reproducible analysis. As a first step for metaQuantome analysis, metaproteomics 
+data needs to be made compatible for subsequent analysis. With this in mind, we have developed a metaQuantome data 
+generation workflow tutorial that ill help users generate inputs for metaQuantome analysis.
+
+![Workflow](../../images/metaquantomeworkflow.png){: width="75%"}
+
+To demonstrate the use of the data creation workflow, we have used a thermophilic biogas reactor dataset wherein municipal 
+food waste and manure is digested to generate methane gas. After one round in the reactor, the microbial community was 
+simplified and enriched via serial dilution. This inoculum was then transferred to a solution of cellulose from Norwegian 
+Spruce and incubated at 65°C. Triplicate samples were taken in a time series from 0 to 43 hours after inoculation and mass 
+spectrometry data was acquired on a Q-Exactive (Thermo) mass spectrometer. For this training, we have chosen two time points-8 hour and 33 hour.
+
+![Dataset](../../images/biogasdataset.png){: width="75%"}
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# **Pretreatments**
+
+The first step in a tutorial is to get the data from the zenodo link provided and making sure that it is in the correct format.
+
+## *Get data*
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files ( 6 MZML files, a Protein FASTA file and Experimental Design file) from [Zenodo]({{ page.zenodo_link }}) 
```

Comment:
```suggestion
> 2. Import the files: 6 MZML files (format=`mzml`, a Protein FASTA file (format=`fasta`), and an Experimental Design file (format=`tabular`) from [Zenodo]({{ page.zenodo_link }}) 
```

Reply:
I suggest specifically confirming what the datatypes/formats of the files should be.